README.md

```diff
@@ -74,17 +74,17 @@ files:
 
 ### CSV Data Files
 
-One CSV file can only store one type of vertex or edge. Vertices and edges of different schema should be stored in different files.
+One CSV file can only store one type of vertex or edge. Vertices and edges of the different schema should be stored in different files.
 
 * `path`: **Required**. Specifies the path where the CSV data file is stored. If a relative path is used, the path and directory of the current configuration file are spliced.
```

Comment:
I don't quite understand this part. It's a little tricky. Please check with developers.